MCP_Server/server.py

```
@@ -390,6 +396,81 @@ def set_clip_name(ctx: Context, track_index: int, clip_index: int, name: str) ->
         logger.error(f"Error setting clip name: {str(e)}")
         return f"Error setting clip name: {str(e)}"
 
+@mcp.tool()
+def duplicate_clip(
+    ctx: Context, 
```

Ruff (0.14.6)

Unused function argument: `ctx`

(ARG001)